pages/Animal-Growth-Accelerator.md

```diff
@@ -7,4 +7,4 @@ It runs on [Organic Food](https://github.com/Slimefun/Slimefun4/wiki/Miscellaneo
 There is only one tier of the Animal Growth Accelerator and it has the following properties:  
```

Comment:
What does it exactly do? Provide a description of what it does.

Reply:
There is a description provided at the top line of this file
> The Animal Growth Accelerator is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to speed up the growth of baby animals.